Docker.DotNet/Endpoints/MiscellaneousOperations.cs

```diff
@@ -21,18 +21,18 @@ public Task AuthenticateAsync(AuthConfigParameters authConfig)
         {
             if (authConfig == null)
             {
-                throw new ArgumentNullException("authConfig");
+                throw new ArgumentNullException(nameof(authConfig));
             }
             var data = new JsonRequestContent<AuthConfigParameters>(authConfig, this.Client.JsonSerializer);
 
-            const string path = "auth";
-            return this.Client.MakeRequestAsync(this.Client.NoErrorHandlers, HttpMethod.Post, path, null, data);
+            const string Path = "auth";
```

Comment:
Why declare the const at all? We could just put it in MakeRequestAsync and have the same "const" without an extra assignment correct?